src/HealthChecks.AzureStorage/BlobStorageHealthCheckOptions.cs

```diff
@@ -0,0 +1,16 @@
+namespace HealthChecks.AzureStorage;
+
+/// <summary>
+/// Represents a collection of settings that configure Azure Storage Blob Service health checks.
```

Comment:
```suggestion
/// Represents a collection of settings that configure <see cref="AzureBlobStorageHealthCheck">Azure Storage Blob Service health check</see>.
```

The same suggestion for other options.